src/main/java/com/bakdata/kafka/ErrorCapturingFlatValueTransformer.java

```diff
@@ -89,6 +91,55 @@
         return new ErrorCapturingFlatValueTransformer<>(transformer, errorFilter);
     }
 
+    /**
+     * Wrap a {@code ValueTransformerSupplier} and capture thrown exceptions. Recoverable Kafka exceptions such as a
+     * schema registry timeout are forwarded and not captured.
+     *
+     * @param supplier {@code ValueTransformerSupplier} whose exceptions should be captured
+     * @param <V> type of input values
+     * @param <VR> type of output values
+     * @return {@code ValueTransformerSupplier}
+     * @see #captureErrors(ValueTransformerSupplier, Predicate)
+     * @see ErrorUtil#isRecoverable(Exception)
+     */
+    public static <V, VR> ValueTransformerSupplier<V, Iterable<ProcessedValue<V, VR>>> captureErrors(
+            final ValueTransformerSupplier<? super V, ? extends Iterable<VR>> supplier) {
```

Comment:
`@NonNull` missing. Also applies to all other `captureErrors` methods.

Reply:
Done